utils/oss.py

```diff
@@ -0,0 +1,122 @@
+# This code is part of Ansible, but is an independent component.
+# This file identifies and gains playbook params, and provides this params to ansible module ecs. 
+# This file implements connection between ansible and Alicloud ecs api via footmark.
+#
+# Copyright (c), xiao zhu <heguimin36@163.com.com>, 2016.08
+# All rights reserved.
+
+import os
+
+try:
+    import footmark.oss
+except ImportError:
+    raise ImportError('footmark is required for the module')
+
+
+class AnsibleACSError(Exception):
+    pass
+
+
+def acs_common_argument_spec():
+    return dict(
+        alicloud_access_key=dict(aliases=['acs_access_key', 'ecs_access_key', 'access_key']),
+        alicloud_secret_key=dict(aliases=['acs_secret_access_key', 'ecs_secret_key', 'secret_key']),
+    )
+
+
+def oss_bucket_argument_spec():
+    spec = acs_common_argument_spec()
+    spec.update(
+        dict(
+            alicloud_region=dict(aliases=['acs_region', 'ecs_region', 'region']),
+            bucket=dict(aliases=['bucket_name', 'name'], type='str', required='True')
+        )
+    )
+    return spec
+
+
+def get_oss_connection_info(module):
+    '''
+    Check module args for credentials, then check environment vars access_key
+    '''
+    access_key = module.params.get('alicloud_access_key')
+    secret_key = module.params.get('alicloud_secret_key')
+    region = module.params.get('alicloud_region')
+
+    if not access_key:
+        if 'ALICLOUD_ACCESS_KEY' in os.environ:
+            access_key = os.environ['ALICLOUD_ACCESS_KEY']
+        elif 'ACS_ACCESS_KEY_ID' in os.environ:
+            access_key = os.environ['ACS_ACCESS_KEY_ID']
+        elif 'ACS_ACCESS_KEY' in os.environ:
+            access_key = os.environ['ACS_ACCESS_KEY']
+        elif 'ECS_ACCESS_KEY' in os.environ:
+            access_key = os.environ['ECS_ACCESS_KEY']
+        else:
+            # in case access_key came in as empty string
+            module.fail_json(msg="access key is required")
+
+    if not secret_key:
+        if 'ALICLOUD_SECRET_ACCESS_KEY' in os.environ:
+            secret_key = os.environ['ALICLOUD_SECRET_ACCESS_KEY']
+        elif 'ACS_SECRET_ACCESS_KEY' in os.environ:
```

Comment:
Please change 'ALICLOUD_SECRET_ACCESS_KEY' to 'ALICLOUD_SECRET_KEY' in the line 62.